PublicSDKAPITester/RCPurchasesAPI.m

```diff
@@ -132,4 +135,13 @@ + (void)checkEnums {
     l = RCLogLevelError;
 }
 
++ (void)checkConstants {
+    double vn = PurchasesVersionNumber;
+    char vs = PurchasesVersionString;
+    NSErrorDomain bed = RCBackendErrorCodeDomain; // TODO does this replace RCBackendErrorDomain?
+    NSErrorDomain ped = RCPurchasesErrorCodeDomain; // TODO does this replace RCPurchasesErrorDomain?
+    //    NSErrorUserInfoKey fk = RCFinishableKey; // TODO where'd this go
```

Comment:
`RCErrorDetails.RCFinishableKey`
`RCErrorDetails.RCReadableErrorCodeKey`
ObjC -> Swift can't do global vars that work across Objc/Swift boundaries 😢